package.json

```diff
@@ -13,10 +13,35 @@
     "parser": "babel-eslint"
   },
   "dependencies": {
+    "@conveyal/lonlat": "^1.1.0",
+    "@mapbox/polyline": "^0.2.0",
+    "@opentripplanner/base-map": "^0.0.21",
```

Comment:
All of these should not be imported like this. We should just import otp-react-redux which will have all of these dependencies.

Reply:
Thanks for pointing that out. I will update when OTP-RR is released (pending https://github.com/opentripplanner/otp-react-redux/pull/150).

Reply:
Dependencies are updated as of commit f81f1c1e2bfb9cfe2da64f59c45f2b1f7fb2b4d1.